CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Unreleased]
+### Added
+- `vars/Fedora.yml`, `tasks/Fedora/main.yml`: Add Fedora as a client OS. Tested on Fedora 25, but should work on newer releases as well.
```

Comment:
I would actually re-word this a bit to say tested on fedora27 since you were able to get the build to pass (which is excellent btw!).

Reply:
Ah, forgot to update the changelog... I'll fix that!